Fix #184: Document tool.{sarifLoggerVersion,language}

lgolding opened the pull request:
@michaelcfanning

Commit:
Fix #184: Document tool.{sarifLoggerVersion,language}

michaelcfanning commented on document/CoreFormat.mdk:
```diff
@@ -649,6 +649,49 @@ If the operating system does not provide such a value, the `fileVersion` propert
+then the `tool` object should contain a property named `sarifLoggerVersion`, whose value is
+a string specifying the version of the logging component.
+
+If this property is present, then it shall contain the Semantic Version 2.0.0 version
```
ok, just a nit, but here's a place I don't think we have to encourage people to use semver. The logging component version should be sufficient to uniquely identify the implementation that produced the log file. That's it. We don't need to encourage standardizing on any particular format because we don't expect mgmt systems to process this data in a uniform way, across multiple tools, etc. This information exists primarily for this scenario: someone has observed a bug in the SARIF output and needs to know what code base produced it.

Commit:
Address PR feedback on tool.sarifLoggerVersion

Don't suggest Semantic Version. The NOTE explains that the
version just needs to be precise enough to allow the logging
component author to uniquely identify it.
